flow/size.yaml

```diff
@@ -62,3 +83,111 @@ components:
           type: integer
           default: 1518
           x-field-uid: 2
+    Flow.SizeWeightPairs:
+      type: object
+      description: |-
+        Frame size distribution, defined as <size, weight> pairs (including IMIX distribution).
+      properties:
+        choice:
+          type: string
+          default: predefined
+          x-field-uid: 1
+          x-enum:
+            predefined:
+              x-field-uid: 1
+            custom:
+              x-field-uid: 2
+        predefined:
```

Comment:
Few questions about predefined distribution:
- which ones are vendor neutral ?
- do we have a link or doc backing what each distribution looks like ?

IMHO, need to understand what value would adding predefined distribution add ? Since we can cover these using custom.